projects/RPi/packages/linux/modprobe.d/brcmfmac.conf

```diff
@@ -0,0 +1,5 @@
+# Disable SAE authentication offload until it works with iwd
+#
+# Defined in drivers/net/wireless/broadcom/brcm80211/include/brcmu_wifi.h
+# as WPA3_AUTH_SAE_PSK 0x40000 /* SAE with 4-way handshake */
```

Comment:
This comment seems wrong, brcmfmac features are defined in https://github.com/raspberrypi/linux/blob/rpi-6.6.y/drivers/net/wireless/broadcom/brcm80211/brcmfmac/feature.h and 0x40000 is SAE_EXT in the downstream RPi kernel (which patched in that feature in the middle of the features list instead of adding it to the end so this differs from upstream...)

see also /sys/kernel/debug/ieee80211/phy0/features

Reply:
Correct! I've removed the redundant comment.